state/protocol/badger/mutator.go

```diff
@@ -894,25 +878,21 @@ func (m *FollowerState) epochStatus(block *flow.Header) (*flow.EpochStatus, bool
 //    this method returns a slice of Badger operations to apply while storing the block.
 //    This includes an operation to index the epoch status for every block, and
 //    operations to insert service events for blocks that include them.
-//  * insertingCandidateTriggersEpochFallback - if inserting this block causes epoch
-//    fallback mode to be triggered, this is true. This causes flag to be set in the
-//    database and a protocol event to be emitted. This can only be true for at most
-//    one block for a given database instance.
 //
 // No errors are expected during normal operation.
```

Comment:
I think comment section for this function needs to be revisited after implementing QC front-loading, lets place a TODO for it.

Reply:
https://github.com/onflow/flow-go/pull/2969/commits/317b9f2605660a45fd3a0e824bea1685ee1e029a